mailpoet/tests/integration/Cron/Workers/SendingQueueBodyCleanupTest.php

```
@@ -0,0 +1,91 @@
+use MailPoet\Settings\SettingsController;
+use MailPoetVendor\Carbon\Carbon;
+
+class SendingQueueBodyCleanupTest extends \MailPoetTest {
+  /** @var SendingQueueBodyCleanup */
+  private $worker;
+
+  /** @var SettingsController */
+  private $settings;
+
+  public function _before() {
+    parent::_before();
```

PHPStan (2.1.46)

[error] Call to an undefined static method MailPoetTest::_before().


(staticMethod.notFound)